admin_guide/high_availability.adoc

```diff
@@ -252,16 +261,71 @@ ip addr show $interface | grep -i MULTICAST
 echo "Check multicast groups ... "
 ip maddr show $interface | grep 224.0.0 | grep $interface
 
-echo "Optionally, add accept rule and persist it ... "
-sudo /sbin/iptables -I INPUT -i $interface -d 224.0.0.18/32 -j ACCEPT
-
-echo "Please ensure the above rule is added on system restarts."
-
 EOF
 done;
 ----
 ====
 
+[[options-environment-variables]]
+=== Command Line Options and Environment Variables
+
+.Command Line Options and Environment Variables
+[cols="1a,3a,1a,4a",options="header"]
+|===
+
+| Option | Variable Name | Default | Notes
+
+|--watch-port
+|`*OPENSHIFT_HA_MONITOR_PORT*`
+|80
+|This port must be open for the service to be running.
+
+|--interface
+|`*OPENSHIFT_HA_NETWORK_INTERFACE*`
+|
+|By default all NICs are watched.
+
+|
+|`*OPENSHIFT_HA_REPLICA_COUNT*`
+|2
+|This must match spec.replicas in the DC
+
+|--virtual-ips
+|`*OPENSHIFT_HA_VIRTUAL_IPS*`
+|
+|E.g., 1.2.3.4-6,1.2.3.9
+
+|--vrrp-id-offset
+|`*OPENSHIFT_HA_VRRP_ID_OFFSET*`
+|0
+|When multiple *ipfailover* deployments are used each must have a unique offset.
+
+|--iptables-chain
+|`*OPENSHIFT_HA_IPTABLES_CHAIN*`
+|INPUT
+|If the chain does not exist it is not created.
```

Comment:
"The name of the iptables chain to automatically add an iptables rule to allow the VRRP traffic on.  If the value is not set, an iptables rule will not be added."